index.bs

```diff
@@ -408,14 +408,14 @@ Excluding inputs generally include
 However, an event whose only effect is to begin or update a fling or scroll
 gesture is not an excluding input.
 
-The user agent may delay the reporting of layout shifts after a
-<a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event
-until such time as it is known that the event does not begin a fling or scroll
-gesture.
-
-The <a href="https://www.w3.org/TR/uievents/#event-type-mousemove">mousemove</a> and
-<a href="https://www.w3.org/TR/pointerevents/#the-pointermove-event">pointermove</a>
-events are also not excluding inputs.
+* When a <a href="https://www.w3.org/TR/pointerevents/#the-pointerdown-event">pointerdown</a> event or
+a <a href="https://www.w3.org/TR/uievents/#event-type-mousedown">mousedown</a> event is received, the
```

Comment:
I think that markdown doesn't like if you don't indent this line within the bullet point

Reply:
ACK.